tests/meta-mender-ci/recipes-testing/mender-artifact-coverage/mender-artifact_%.bbappend

```diff
@@ -0,0 +1,62 @@
+SUMMARY_append_mender-testing-enabled = ": Build mender-artifact with coverage recording enabled"
+
+DEPENDS_append_mender-testing-enabled = " gobinarycoverage-native rsync-native"
+
+do_instrument_mender-artifact[dirs] =+ "${GOTMPDIR}"
+do_instrument_mender-artifact[doc] = "Modifies the mender-artifact source to enable coverage analysis"
+do_instrument_mender-artifact () {
+    cd ${B}/src/${GO_IMPORT}
+    bbwarn "Building the mender-artifact tool with coverage analysis"
+    oe_runmake instrument-binary
+}
+
+do_configure_prepend_mender-testing-enabled () {
+    # Remove all the src present in build if it is not a symbolic link to ${S}
+    if [ -d ${B}src ]; then
+        rm -rf ${B}src
+    fi
+
+}
+
+do_configure_append_mender-testing-enabled () {
+    # Remove the symbolic link created by go.bbclass in do_configure
+    if [ -h ${B}src ]; then
+        rm ${B}src
+    fi
+    mkdir -p ${B}src/${GO_IMPORT}
+    rsync --archive --recursive --delete ${S}/src/${GO_IMPORT}/ ${B}/src/${GO_IMPORT}/
+}
+
+python () {
+    if bb.utils.contains("DISTRO_FEATURES", "mender-testing-enabled", True, False, d):
```

Comment:
This is called `MENDER_FEATURES` now. The Mender client recipe has the correct name, which makes me wonder: You didn't copy this from an old branch, did you?